### PR TITLE
Fix linting errors of sonarcloud

### PR DESCRIPTION
**Pull request by adrianrobotka**
No functional changes, just fix linting errors.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -14,6 +14,9 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```
Comment (rlacko58) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: This is a good Idea, it should provide some better security and performance

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +33,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (rlacko58) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:36]: Someone forgot to throw this

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -30,18 +33,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (rlacko58) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:46]: Some left this typo in the code, it should be alright like this

**Review by rlacko58 — CHANGES_REQUESTED** [review]
I have written some of my toughts about your changes, please review them.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (adrianrobotka) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: Thanks. It's good enough for this game.

**Pull request merged**